e2e.sh

```diff
@@ -2,23 +2,29 @@
 
 set -Eeuxo pipefail
 
+cluster=kind-1
 
 
 run_kind() {
-    echo "Download kind binary..."
-    wget -O kind 'https://docs.google.com/uc?export=download&id=1-oy-ui0ZE_T3Fglz1c8ZgnW8U-A4yS8u' --no-check-certificate && chmod +x kind && sudo mv kind /usr/local/bin/
+    if [ ! -x /usr/local/bin/kind ]; then
+        echo "Download kind binary..."
+        wget -O kind 'https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-linux-amd64' --no-check-certificate && chmod +x kind && sudo mv kind /usr/local/bin/
```

Comment:
Do we really need `--no-check-certificate`?

Seems weird that we skip here, but not below.